src/CoreShop2VueStorefrontBundle/Bridge/DocumentMapper/DocumentCategoryMapper.php

```diff
@@ -96,6 +94,11 @@ public function mapToDocument($category, ?string $language = null): Category
         return $esCategory;
     }
 
+    protected function getDocumentClass(): string
```

Comment:
This likely needs to be public to allow `PersisterFactory` to resolve them upfront.